src/goto-symex/goto_symex.h

```diff
@@ -232,12 +232,15 @@ class goto_symext
 
   virtual void symex_assume(statet &, const exprt &cond);
 
-  // gotos
+  /// Merge all branches joining at the current program point at the current
+  /// program point. It will apply \ref merge_goto for each goto state (each of
```

Comment:
The end of the first sentence is repeated.

Reply:
Oops, thanks, fixed.